include/cupla/config/AnyOmp5.hpp

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2019 Rene Widera
```

Comment:
This should probably be updated?

Reply:
Yes and no: I actually only renamed this file and changed the alpaka include from omp4 to omp5. The file `AnyOacc.hpp` is the one which is actually new, although it is only a copy of the old `CpuOmp4.hpp` with the alpaka include changed. I changed year and author in `AnyOacc.hpp` just now, which has the advantage, that git recognizes this one as new and  `AnyOmp5.hpp` as renamed.